libs/design/src/scss/daff-typography.scss

```diff
@@ -7,7 +7,7 @@ body,html {
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
 
-    @include breakpoint(large) {
+    @include breakpoint(mobile) {
```

Comment:
I think you accidentally changed this to mobile instead of big-tablet.

Reply:
Nope, I changed it to mobile on purpose.